src/topology/basic.lean

```diff
@@ -1184,6 +1195,20 @@ lemma continuous_iff_ultrafilter {f : α → β} :
   continuous f ↔ ∀ x (g : ultrafilter α), ↑g ≤ 𝓝 x → tendsto f g (𝓝 (f x)) :=
 by simp only [continuous_iff_continuous_at, continuous_at_iff_ultrafilter]
 
+lemma continuous.closure_preimage_subset {f : α → β}
+  (hf : continuous f) (t : set β) :
+  closure (f ⁻¹' t) ⊆ f ⁻¹' (closure t) :=
+begin
+  rw ← (is_closed_closure.preimage hf).closure_eq,
+  mono*,
+  exact subset_closure,
```

Comment:
`show_term` suggests the following, which isn't much less readable and makes it more obvious which lemmas are used.
```suggestion
  exact closure_mono (preimage_mono subset_closure),
```

Reply:
I like one less line in Eric's suggestion. Should I click "commit suggestion" or "resolve conversation" or possibly both? I ask because (1) I am too new to have a strong opinion about the style, and (2) I worry that this will start lengthy checks again.

Reply:
Clicking "commit suggestion" will apply the change _and_ resolve the conversation. Clicking "resolve conversation" is indicating "no, I don't want to do this.

The lengthly checks you describe are only about 40 minutes, which is probably less time than the time it takes for another reviewer to see this - but it's also fine to wait for a second opinion :)

Reply:
For the record, I think shortening these lines is worth it. But I'm not so familiar with this part of the library as to really do the review.

Reply:
I wrote this `mono*` line on Zulip for pedagogical purposes, to emphasize that we have a tactic handling those monotonicity proofs, but in mathlib we can use the expanded version, this will save a bit of compiling time (no much I think, but still something).

Reply:
Ok, I'll "commit suggestion"!